config/config.go

```diff
@@ -172,9 +180,18 @@ func CreateServer(args []string, funcs ...Option) (srv *server.Server) {
 		runtime.GOMAXPROCS(*goMaxProcess)
 	}
 
+	if *prometheusBind != "" {
+		prometheusmetrics.New(
+			prometheusmetrics.WithAddr(*prometheusBind),
+			prometheusmetrics.WithPath(*prometheusPath),
+			prometheusmetrics.WithLogger(logger),
+		).Run()
+	}
```

Comment:
I made changes to the naming convention, mainly prefixed with prometheus instead of metrics. 

I would not prefer to have `.Run()` left hanging here. Instead I am keen to make it managed by `Server`.  Prometheus should start after server (main server, imagor app, libvips) is ready.

Reply:
Cool! I moved `PrometheusMetrics.Run()` into `server.RunContext()`

Reply:
9621a48056127e113063d31421c029ee9243f027